### ISSUE TITLE
Which settings were used for NiftyReg?

### ISSUE THREAD

**Issue by veritas9872** [voxelmorph/voxelmorph#238]
Hello, I have found other issues asking for the exact settings for the ANTs package and I was curious if the script for generating NiftyReg registrations could be shared as well.

**Comment by adalca** [comment]
@veritas9872 ,

> I have found other issues asking for the exact settings for the ANTs package 

What did you ask about the ANTs package that you did not get answered? In the course of our registration work more broadly, we ran pretty thorough searches over parameters, and included these in the papers. For example, bottom of page 7 of https://arxiv.org/pdf/1809.05231.pdf. The same with NiftyReg -- we made only tiny modifications to the default parameters to improve accuracy a bit. For example in the linked pdf, we only modified the cost function (CC instead of NMI) grid spacing of 5, and 500 iterations. 

> if the script for generating NiftyReg registrations could be shared as well

We have several registration publications where we may have tuned niftyReg differently depending on the task -- for example in multi-modal registration @mu40 used default NiftyReg parameters because those seemed to work well for that task (https://arxiv.org/pdf/2004.10282.pdf). In terms of running, i think just something like 

```reg_f3d -ref atlas.nii -flo vol.nii -res result.nii -cpp result_cpp.nii``` or `reg_aladin ` as appropriate, see http://cmictig.cs.ucl.ac.uk/wiki/index.php/Reg_f3d and http://cmictig.cs.ucl.ac.uk/wiki/index.php/Reg_aladin

Should work.
